commands/project/repo.go

```diff
@@ -29,5 +55,7 @@ func NewCmdRepo(f *cmdutils.Factory) *cobra.Command {
 	repoCmd.AddCommand(repoCmdFork.NewCmdFork(f, nil))
 	repoCmd.AddCommand(repoCmdSearch.NewCmdSearch(f))
 
+	repoCmd.Flags().BoolVarP(&opts.OpenInBrowser, "web", "w", false, "Open repo in a browser. Uses default browser or browser specified in BROWSER variable")
```

Comment:
can't it be a `BoolVar` instead and instead of creating a struct one just does `repoCmd.Flags().Changed("web")` ?